src/components/pages/Header.jsx

```diff
@@ -0,0 +1,97 @@
+
+    const handleToggle = () => {
+        setIsOpen(!isOpen);
+    };
+    
+    const location = useLocation();
+    useEffect(() => {
+        if (location.pathname === "/search") {
+            document.querySelector('form').style.visibility = "hidden";
+        } else{
+            document.querySelector('form').style.visibility = "visible";
+        }
+        
+        if(location.pathname === "/login") {
+            document.querySelector('#btnLogin').style.visibility = "hidden";
+        } else {
+            document.querySelector('#btnLogin').style.visibility = "visible";
+        }
+        
+        if(location.pathname === "/register") {
+            document.querySelector('#btnJoin').style.display = "none";
+        } else {
+            document.querySelector('#btnJoin').style.display = "block";
+        }
+    }, [location.pathname]);
+
+    const navigate = useNavigate();
+    const search = () => {
+        navigate("/search");
+    };
+    const login = () => {
+        navigate("/login");
+    };
+    const register = () => {
+        navigate("/register");
+    };
+
+    return (
+        <header className="h-[80px] w-full flex items-center px-11 justify-between">
+            <Link className="text-black font-spartan font-semibold text-xl sm:text-3xl w-1/3" to="/">TalentStudio</Link>
+            {}
+            {/* Nav for the big screens */}
+            <nav className="w-2/3 hidden relative py-6 sm:flex justify-between">
+                <form className="border-2 border-red rounded-full h-10 w-2/4 flex justify-between items-center px-4 ml-6 gap-1">
+                    <input className='focus:outline-none w-2/3' placeholder="Search online classes or teachers..." />
+                    <button onClick={search}><img src={searchIcon} className="h-[24px]" /></button>
+                </form>
+                <div id="mainNav" className="flex gap-4 items-center">
+                    <button id="btnLogin" className="font-spartan font-semibold text-base sm:text-lg flex items-center "
+                        onClick={login}>
+                        <img src={loginIcon} className="h-[15px] sm:h-[20px]" />
+                        Log In
+                    </button>
+                    <button id="btnJoin" className='bg-red hover:bg-pureWhite hover:text-red w-16 h-8 hover:border-2 hover:border-red text-white font-spartan font-semibold text-base sm:text-lg rounded-lg transition duration-300 easy-in'
+                        onClick={register}>Join
+                    </button>
+                </div>
+            </nav>
+
+            {/* Nav for the small screens */}
+            
+            <nav className="sm:hidden relative flex flex-row my-4 gap-2">
+                <button onClick={search}><img src={searchIcon} className="h-[24px]" /></button>
+                <button onClick={handleToggle}>{isOpen ? <img src={closeIcon} className="h-[24px]" /> : <img src={navIcon} className="h-[24px]" />}</button>
```

Comment:
you can use Tailwind property h-6 for 24 px

<img width="493" alt="image" src="https://github.com/user-attachments/assets/a357b60e-daa0-4048-994c-e6d87d0224de">

Reply:
Thank you, Anna! I will fix this.